backend/models/lesson.model.ts

```diff
@@ -0,0 +1,36 @@
+import mongoose, { Schema, Document } from "mongoose";
+
+export interface Lesson extends Document {
+  id: string;
+  course: string;
+  title: string;
+  description: string;
+  image: string;
+  content: [Object];
+}
+
+const LessonSchema: Schema = new Schema({
+  course: {
+    type: String,
+    required: true,
+  },
+  title: {
+    type: String,
+    required: true,
+  },
+  description: {
+    type: String,
+    required: false,
```

Comment:
Hmm consistency thing - is there a preference between not including `required` attribute vs. specifying `required: false`?